modules/airgap/setup/docker_ops.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -x
+echo "$@"
+
+# Define the input file
+product=${1}
+hostdns=${2}
+username=${3}
+password=${4}
+
+IFS=$'\n' # set the Internal Field Separator to newline
+
+image_files=`ls $product*.txt`
+image_files=$(echo $image_files | tr " " "\n")
+echo $image_files
```

Comment:
should we double quotes the var as best practice ?

Reply:
to no paste the link in various places, if you are going to change that, please take a look in all scripts, i see a lot of vars not double quoted

Reply:
Updated, added shellcheck